__docs__/wonder-blocks-form/labeled-text-field.stories.tsx

```diff
@@ -500,26 +499,19 @@ export const ErrorLight: StoryComponentType = (args: any) => {
     };
 
     return (
-        <View style={styles.darkBackground}>
-            <LabeledTextField
-                {...args}
-                label={
-                    <LabelMedium style={styles.whiteColor}>Email</LabelMedium>
-                }
-                description={
-                    <LabelSmall style={styles.offWhiteColor}>
-                        Please provide your personal email
-                    </LabelSmall>
-                }
-                type="email"
-                value={value}
-                light={true}
-                onChange={setValue}
-                placeholder="Email"
-                validate={validate}
-                onKeyDown={handleKeyDown}
-            />
-        </View>
+        <LabeledTextField
+            {...args}
+            label="Email"
+            description="Please provide your personal email"
+            type="email"
+            value={value}
+            light={true}
+            onChange={setValue}
+            placeholder="Email"
+            validate={validate}
+            onKeyDown={handleKeyDown}
+            required={true}
```

Comment:
Making the light `LabelledTextField` stories have `required=true` so that we can include that case in visual regression tests (the required indicator styling wasn't correct in the [previous version](https://khan.github.io/wonder-blocks/?path=/story/packages-form-labeledtextfield--light&args=required:!true). This styling issue was addressed by passing in a string instead for the label prop)